apps/web/src/pages/Applications/ApplicationEducationPage/utils.tsx

```diff
@@ -97,7 +96,7 @@ const appliedWorkListMessages = (isIAP = false) => [
 export const getEducationRequirementOptions = (
   intl: IntlShape,
   locale: Locales,
-  classificationGroup?: ClassificationGroup,
+  classificationGroup?: string,
```

Comment:
Since classification is a user generated string, we were required to cast everything to this type when we wanted to use it so it was extra code that didn't actually guard anything since it was all being cast anyway. Changing this to string, while weaker, is just being more honest about the type :sweat_smile:

Reply:
Yeah, that's fair.  :+1: